indexers/vector/PysparnnIndexer/requirements.txt

```diff
@@ -0,0 +1,3 @@
+jina
+numpy
```

Comment:
numpy is a `jina` core dependency, so there is no point on adding it here. also `jina` is added in the `dockerfile`